apps/wpcom-block-editor/src/calypso/features/iframe-bridge-server.js

```diff
@@ -848,6 +848,23 @@ function handleUncaughtErrors( calypsoPort ) {
 	};
 }
 
+function handleEditorLoaded( calypsoPort ) {
+	const unsubscribe = subscribe( () => {
+		const isReady = select( 'core/editor' ).__unstableIsEditorReady();
```

Comment:
Will this run inside wp-admin? Note that sometimes Atomic sites run older version of Gutenberg, so you should test that `__unstableIsEditorReady` exists.

Reply:
Thanks, changed